packages/blade/src/components/Card/_decisions/decisions.md

```diff
@@ -0,0 +1,512 @@
+    - Secondary action button
+
+
+## Card API Constraints 
+
+**Header Leading Section**
+
+- `Title` & `Subtitle`
+  - Size & text variant cannot be changed
+- `Icon`
+  - Size & color cannot be changed
+- `Counter`
+  - Size can be changed but not recommended
+  - Variant can be changed
+
+**Header Trailing Section**
+
+Anything can be changed in these components, but whats given out of the box are the recommended: 
+
+Only one of the component can be used at a time: 
+- Badge
+- Text
+- Link
+- IconButton - only tertiary variant
+
+Other recommendations from design side: 
+
+- Never use a primary/secondary action in the header, footer are meant for such actions.
+  - This means it's not recommended to change variant of IconButton
+
+**Footer Leading Section**
+
+- `Title` & `Subtitle`
+  - Size & text variant cannot be changed
+
+**Footer Trailing Sections**
+
+- Only two action buttons can be present `primary` or `secondary`
+- Users can totally omit action buttons
+- They can also use either variants of buttons `primary` or `secondary`
+
+## API
+
+With the above constraints we can propose the Card API as follows: 
+
+Sample usage:
+
+```jsx
+<Card backgroundLevel={1}>
+  <Card.Header>
+    <Card.HeaderLeading  
+      title="Payments Links" 
+      subtitle="Share payment link via an email, SMS, messenger, chatbot etc." 
+      prefix={<CardHeaderIcon icon={DollarIcon} />}
+      suffix={<CardHeaderCounter amount={20} />}
+    />
+    <Card.HeaderTrailing 
+      visual={<CardHeaderBadge variant="positive">NEW</CardHeaderBadge>} 
+    />
+  </Card.Header>
+  <Card.Body>
+    Card Body Content
+  </Card.Body>
+  <Card.Footer>
+    <Card.FooterLeading 
+      title="Card Footer Title" 
+      subtitle="Card footer subtitle" 
+    />
+    <Card.FooterTrailing 
+      actions={{
+        primaryAction: { text: 'Know more', onClick: () => {} },
+        secondaryAction: { text: 'Read Docs', onClick: () => {} },
+      }}
+    />
+  </Card.Footer>
+</Card>
+```
+
+
+### `Card` API
+
+
+| Prop              | Type   | Default | Description                                                                            | Required |
+| ----------------- | ------ | ------- | -------------------------------------------------------------------------------------- | -------- |
+| `surfaceLevel` | `2, 3` | `2`     | Surface level of the card background color, use this based on where the card is placed |          |
+
+
+### `Card.HeaderLeading` API
+
+
+| Prop             | Type              | Default     | Description                                              | Required |
+| ---------------- | ----------------- | ----------- | -------------------------------------------------------- | -------- |
+| `title`          | `string`          | `undefined` | Title of the Card                                        | ✅        |
+| `subtitle`       | `string`          | `undefined` | Subtitle of the Card                                     |          |
```

Comment:
In Alert we're using `description`. Should we follow similar convention here?

Reply:
Discussed about this with Rastogi. 

Conclusion: 

subtitle & description conveys different context based on different components. 

In Alert description the the body text which can be long form content that's why contextually the naming `description` makes sense
But in CardHeader `subtitle` makes more sense since it will not have any long form content more than 1 or 2 lines.